RecoTracker/DisplacedRegionalTracking/interface/DisplacedVertexCluster.h

```diff
@@ -0,0 +1,66 @@
+#ifndef RecoTracker_DisplacedRegionalTracking_DisplacedVertexCluster_h
+#define RecoTracker_DisplacedRegionalTracking_DisplacedVertexCluster_h
+
+#include <cmath>
+#include <limits>
+#include <list>
+#include <utility>
+
+#include "DataFormats/Common/interface/View.h"
+#include "DataFormats/Candidate/interface/VertexCompositeCandidate.h"
+#include "DataFormats/Math/interface/Vector3D.h"
+
+class DisplacedVertexCluster;
+typedef std::list<DisplacedVertexCluster>::iterator DisplacedVertexClusterItr;
+
+class DisplacedVertexCluster {
+public:
+  static constexpr double kInvalidDouble = std::numeric_limits<double>::quiet_NaN();
+
+  DisplacedVertexCluster()
+      : valid_(false), rParam2_(kInvalidDouble), sumOfCenters_(0.0, 0.0, 0.0), centerOfMass_(0.0, 0.0, 0.0) {}
+
+  DisplacedVertexCluster(const edm::View<reco::VertexCompositeCandidate> &, const unsigned, const double);
+
+  ~DisplacedVertexCluster() { constituents_.clear(); }
```

Comment:
`std::vector` clears itself in its destructor, this could become just
```suggestion
  ~DisplacedVertexCluster() = default;
```
or omitted (in which case the compiler generates the default destructor).